calibrators/intrinsic_camera_calibrator/intrinsic_camera_calibrator/config/ceres_intrinsics_calibrator.yaml

```
@@ -1,5 +1,5 @@
 # yamllint disable-line rule:truthy
-board_type: chess_board
+board_type: dot_board
```

please revert :pray:  this change is just for our experiments
```suggestion
board_type: chess_board
```